src/user/models/__init__.py

```diff
@@ -1 +1,4 @@
 from .user import User
+from .profile_image import ProfileImage
+from .social_link import SocialLink
+from .activation_code import ActivationCode
```

Comment:
lint error
add a blank line to the end of this file to fix this issue

Reply:
I just add a new blank line to the end of the file